rotkehlchen/inquirer.py

```diff
@@ -41,6 +49,8 @@ def world_to_cryptocompare(asset: Asset) -> Asset:
         asset = cast(NonEthTokenBlockchainAsset, 'DATA')
     elif asset == S_IOTA:
         asset = cast(NonEthTokenBlockchainAsset, 'IOT')
+    elif asset == S_NANO and timestamp and timestamp < XRB_NANO_REBRAND_TS:
```

Comment:
should the opposite also be there? So something like:

```python
elif asset == S_RAILBLOCKS and timestamp and timestamp >= XRB_NANO_REBRAND_TS:
    return S_NANO
```

Sidenote: IF this rebranding happens a lot and is not handled by cryptocompare properly could be worth to generalize this ... but that's a big if

Reply:
I think it should stay the way it is... as the Raiblocks (XRB) was the old symbol and NANO is the new one... so we fallback to XRB in case NANO's price wasn't found.

TBH, i am not following all cryptocurrencies for rebranding news so maybe this happened to others... but yea if that's the case, the implementation should be generalized.

Reply:
But what happens if someone inputs it as Railblocks in Rotkehlchen? Shouldn't it also work in that case? Maybe the user is not aware of the rebranding. That's the use-case I was thinking about.

Reply:
So this means that we want 2 if conditions... fallback to XRB from NANO and vice versa. Would you agree?

Reply:
yes that's what I think would make sense

Reply:
Ok, i added the new condition